tests/general/default_config_test.go

```diff
@@ -35,8 +35,9 @@ func TestDefaultGatewayConfig(t *testing.T) {
 		"",
 		func(collector testutils.Collector) testutils.Collector {
 			return collector.WithArgs().WithEnv(map[string]string{
-				"SPLUNK_ACCESS_TOKEN": "not.real",
-				"SPLUNK_REALM":        "not.real",
+				"SPLUNK_NETWORK_INTERFACE": "0.0.0.0",
```

Comment:
I think this will help confirm the default behavior doesn't now require this env var, but may be missing something.

```suggestion
```

Reply:
ouch this is a bad merge 🤦